.github/dependabot.yml

```diff
@@ -14,3 +14,7 @@ updates:
     directory: "/" # Location of package manifests
     schedule:
       interval: "daily"
+  - package-ecosystem: "pip"
```

Comment:
Probably worth adding github actions to the file now

Reply:
Added github-actions as a package-ecosystem per https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot